rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -315,60 +346,66 @@ def _prep_modules(self, p_id, ro_session):
         if results:
             for row in results:
                 module_name = self._lookup_code_value(row.codeId, ro_session)
-                modules.append({
+                module_data = {
                     'mod_module': module_name,
                     'mod_baseline_module': 1 if module_name in self._baseline_modules else 0,  # Boolean field
                     'mod_authored': row.authored,
                     'mod_created': row.created,
                     'mod_language': row.language,
-                    'mod_status': BQModuleStatusEnum.SUBMITTED.name,
-                    'mod_status_id': BQModuleStatusEnum.SUBMITTED.value,
-                })
+                }
+                # Default status, may be updated based on consent answer
+                module_status = BQModuleStatusEnum.SUBMITTED
 
                 # check if this is a module with consents.
-                if module_name not in _consent_module_question_map:
-                    continue
+                if module_name in _consent_module_question_map:
```

Comment:
Note:  The diff ended up looking more significant than the content changes were.   See the diff from the participant resource generator for comparison.  Logically, the code is very similar to the previous version (except for handling the answer lookup to find the status value to use on line 400).

Reply:
Thanks, that helped alot